### PR TITLE
Add EUI64 Syscall Support 

### PR DESCRIPTION
**Pull request by tyler-potyondy**
This PR adds the corresponding libtock syscall for the EUI64 capsule and a u64 returncode parsing method. 

This PR presents a version compatible with the current state of libtock-c. A version compliant with the libtock rewrite and new formatting has also been committed to the `libtock-folders` branch.

### CODE REVIEW

**Comment by bradjc** [comment]
Can you split the tock.c new function to its own PR?

File: libtock/eui64.c [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include "eui64.h"
+
+returncode_t libtock_eui64_command_getter(uint64_t* eui64) {
```
Comment (bradjc) [inline libtock/eui64.c:3]: -ter

File: libtock/eui64.c [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  // Issue the command to the kernel
+  syscall_return_t cval = command(DRIVER_NUM_EUI64, EUI64_CMD_GETTER, 0, 0);
+  return tock_command_return_u64_to_returncode(cval, eui64);
+}
+
+returncode_t get_eui64(uint64_t *eui64) {
```
Comment (bradjc) [inline libtock/eui64.c:9]: Prefix

File: libtock/eui64.c
Comment (bradjc) [inline libtock/eui64.c:1]: Need exists function.

File: examples/tests/eui64/main.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#include <eui64.h>
+#include <stdio.h>
```
Comment (bradjc) [inline examples/tests/eui64/main.c:2]:
```suggestion
#include <stdio.h>

#include <eui64.h>
```

**Review by bradjc — COMMENTED** [review]
Needs rebase

**Review by bradjc — APPROVED** [review]

**Review by ppannuto — APPROVED** [review]

**Pull request merged**